packages/charts/examples/components/demo/chart-types/area/area-chart-bi-directional-stacked/area-chart-bi-directional-stacked-example.component.ts

```diff
@@ -0,0 +1,153 @@
+import { Component, OnInit } from "@angular/core";
+import {
+    AreaAccessors,
+    areaGrid,
+    AreaRenderer,
+    Chart,
+    ChartAssist,
+    IAreaAccessors,
+    IChartSeries,
+    IXYScales,
+    LinearScale,
+    stackedArea,
+    TimeScale,
+    XYGrid,
+    XYGridConfig
+} from "@nova-ui/charts";
+import moment from "moment/moment";
+
+@Component({
+    selector: "area-chart-bi-directional-stacked-example",
+    templateUrl: "./area-chart-bi-directional-stacked-example.component.html",
+})
+export class AreaChartBiDirectionalStackedExampleComponent implements OnInit {
+    public chartTop: Chart;
```

Comment:
I guess we could get rid of chartTop and chartBottom members since the chart assists keep public references to them.